example/ck_tile/40_streamk_gemm/run_gemm_example.inc

```
@@ -279,8 +275,8 @@ int run_gemm_example_with_layouts(int argc,
               << " B_Type=" << DataTypeTraits<BDataType>::name
               << " C_Type=" << DataTypeTraits<CDataType>::name
               << " reduction_strategy=" << arg_parser.get_str("reduction_strategy") << " "
-              << ave_time << " ms, " << tflops << " TFlops, " << gb_per_sec << " GB/s, "
-              << std::endl;
+              << " persistent_dp=" << arg_parser.get_str("persistent_dp") << " " << ave_time
+              << " ms, " << tflops << " TFlops, " << gb_per_sec << " GB/s, " << std::endl;
 
     bool pass = true;
```

```suggestion
    bool pass = false;
```
nit: I recommend failing by default. It looks like the bool will be assigned later but generally you want the test to deliberately set the flag to true as to mitigate the possibility of silent failures.

---

updated with the suggested changes